src/frontends/tensorflow/src/op/strided_slice.cpp

```diff
@@ -15,28 +15,24 @@ namespace op {
 
 OutputVector translate_strided_slice_op(const NodeContext& node) {
     auto input = node.get_input(0);
-    auto rank = input.get_partial_shape().rank();
     auto begin = node.get_input(1);
     auto end = node.get_input(2);
     auto strides = node.get_input(3);
 
-    auto begin_mask = node.get_attribute<int64_t>("begin_mask");
-    auto end_mask = node.get_attribute<int64_t>("end_mask");
-    auto new_axis_mask = node.get_attribute<int64_t>("new_axis_mask");
-    auto ellipsis_mask = node.get_attribute<int64_t>("ellipsis_mask");
-    auto shrink_axis_mask = node.get_attribute<int64_t>("shrink_axis_mask");
+    auto begin_mask = node.get_attribute<int64_t>("begin_mask", 0);
```

Comment:
nitpick: it looks like these masks should always be there, if they are not, this is a mistake
py API (https://www.tensorflow.org/api_docs/python/tf/strided_slice):
![image](https://user-images.githubusercontent.com/21181928/183352990-8cfc1075-8c37-4840-a7d0-381b8f7fda13.png)
raw ops(https://www.tensorflow.org/api_docs/python/tf/raw_ops/StridedSlice):
![image](https://user-images.githubusercontent.com/21181928/183355232-0b11c782-e660-49c5-9cc1-9b2845543ebf.png)